docs/docs/cmd/spp/model/model-get.mdx

```diff
@@ -0,0 +1,994 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spp model get
+
+Retrieves information about a document understanding model
+
+## Usage
+
+```sh
+m365 spp model get [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --siteUrl <siteUrl>`
+: The URL of the content center site.
+
+`-i, --id [id]`
+: The unique ID of the model to retrieve. Specify `id` or `title` but not both.
+
+`-t, --title [title]`
+: The display name of the model to retrieve. Specify `id` or `title` but not both.
+
+`--withPublications`
+: Retrieves information about the library where it has been applied.
+```
+
+<Global />
+
+## Examples
+
+Retrieve information about a SharePoint Premium document understanding model, using model UniqueId.
+
+```sh
+m365 spp model get --siteUrl "https://contoso.sharepoint.com/sites/ContentCenter" --id "7645e69d-21fb-4a24-a17a-9bdfa7cb63dc"
+```
+
+Retrieve information about a SharePoint Premium document understanding model, using model Title
+
+```sh
+m365 spp model get --siteUrl "https://contoso.sharepoint.com/sites/ContentCenter" --title "climicrosoft365Model"
+```
+
+## Response
+
+### Standard response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
```

Comment:
This response is looking gigantic right now. Let's try to trim it down a bit by:
* For arrays, let's show only 1 object in the array.
* Right now, we have multiple fields in the model like: address, discount percentage, date of change, ... Let's show only one of those.

Reply:
@milanholemans  ok, it is done now, the response should be shorter :)